fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanService.ts

```diff
@@ -398,6 +410,30 @@ export class FlightPlanService<P extends FlightPlanPerformanceData = FlightPlanP
     plan.startAirwayEntry(at);
   }
 
+  public async continueAirwayEntryViaAirway(airway: Airway, planIndex: number, alternate?: boolean): Promise<boolean> {
+    const plan = alternate
+      ? this.flightPlanManager.get(planIndex).alternateFlightPlan
+      : this.flightPlanManager.get(planIndex);
+
+    return await plan.continueAirwayEntryViaAirway(airway);
```

Comment:
Shouldn't need the `await` here? You can also leave it in if you want.

```suggestion
    return plan.continueAirwayEntryViaAirway(airway);
```

Reply:
Good point.